src/store/models.ts

```diff
@@ -36,7 +45,11 @@ export default {
   },
   getters: {
     getModelById: state => (id: number) => {
-      return state.models.find(model => model.id === id);
+      let model = state.models.find(model => model.id === id);
+      if (model === undefined) {
+        model = { name: "???" };
+      }
+      return model;
```

Comment:
I know I added this here - I'm thinking though that the handling of unrecognized models should go in the corresponding components, not the general getter, so that each component may decide how best to deal with it.